### PR TITLE
[WAI] get started revision

### PR DESCRIPTION
**Pull request by deadlypants1973**
PCX-10058

### CODE REVIEW

File: content/workers-ai/get-started/rest-api.md [hunk 1]
```diff
@@ -18,23 +21,26 @@ To create an API token:
 1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com).
 2. Select the user icon on the top right of your dashboard > **My Profile**.
 3. Select [**API Tokens**](https://dash.cloudflare.com/profile/api-tokens) > **Create Token**. 
-4. You can go to **Workers AIs** template > **Use template** or go to **Create Custom Token** > **Get started**.
+4. Go to the **Workers AI** template > select **Use template**. You can choose also go to **Create Custom Token** > **Get started** to customize a token.
```
Comment (marciocloudflare) [inline content/workers-ai/get-started/rest-api.md:24]:
```suggestion
4. Go to the **Workers AI** template > select **Use template**. You can also go to **Create Custom Token** > **Get started** to customize a token.
```

Comment (marciocloudflare) [reply content/workers-ai/get-started/rest-api.md:24]: Not sure what the sentence intent was

File: content/workers-ai/get-started/rest-api.md [hunk 2]
```diff
@@ -18,23 +21,26 @@ To create an API token:
...
+5. Select **Continue to summary** > in Workers AI API token summary, select **Create Token**.
 
 ## 2. Run a model via API
-After creating your token, you can authenticate and make requests to the API using your API token in the request headers. For example, here is an API request to get all deployments in a project.
 
-In this example, we will run the `@cloudflare/meta-llama/llama-2-7b` model:
+After creating your API token, authenticate and make requests to the API using your API token in the request headers. For example, here is an API request to get all deployments in a project.
```
Comment (marciocloudflare) [inline content/workers-ai/get-started/rest-api.md]: The last sentence `For example, here is an API request to get all deployments in a project.` seems to imply we're going to see an example, but then we don't. Is the `In this guide, you will...` below the example mentioned? If so consider restructuring how these 2 sentences flow for improving the clarity of where the example is.

File: content/workers-ai/get-started/workers-wrangler.md [hunk 3]
```diff
@@ -165,10 +167,11 @@ $ npx wrangler deploy
 # Outputs: https://hello-ai.<YOUR_SUBDOMAIN>.workers.dev
 ```
 
-You can now visit the URL to run your AI Worker.
+Your Worker will be deployed to your [`workers.dev`](/workers/configuration/routing/workers-dev/) custom subdomain. You can now visit the URL to run your AI Worker.
 
-By finishing this tutorial, you have created a Worker, connected it to Workers AI, and ran an inference tasks from your model.
+By finishing this tutorial, you have created a Worker, connected it to Workers AI through an AI binding, and ran an inference tasks from your model.
```
Comment (marciocloudflare) [inline content/workers-ai/get-started/workers-wrangler.md:172]:
```suggestion
By finishing this tutorial, you have created a Worker, connected it to Workers AI through an AI binding, and ran an inference task from your model.
```

**Review by marciocloudflare — APPROVED** [review]

File: content/workers-ai/get-started/rest-api.md (hunk 2 above)
Comment (deadlypants1973) [inline content/workers-ai/get-started/rest-api.md]:
```suggestion
After creating your API token, authenticate and make requests to the API using your API token in the request.
```

File: content/workers-ai/get-started/rest-api.md [hunk 4]
```diff
@@ -18,23 +21,26 @@ To create an API token:
 1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com).
 2. Select the user icon on the top right of your dashboard > **My Profile**.
 3. Select [**API Tokens**](https://dash.cloudflare.com/profile/api-tokens) > **Create Token**. 
-4. You can go to **Workers AIs** template > **Use template** or go to **Create Custom Token** > **Get started**.
+4. Go to the **Workers AI** template > select **Use template**. You can choose also go to **Create Custom Token** > **Get started** to customize a token.
+5. Select **Continue to summary** > in Workers AI API token summary, select **Create Token**.
 
 ## 2. Run a model via API
```
Comment (deadlypants1973) [inline content/workers-ai/get-started/rest-api.md:27]: @marciocloudflare start here

**Pull request closed without merging**